exporter/src/pool/bin/main.rs

```diff
@@ -1,36 +1,34 @@
-use std::{env, time::Duration};
-
 use actix_web::{http::header, middleware, web, HttpResponse, HttpServer, Responder};
-use clap::{App, Arg, ArgMatches};
-use prometheus::{Encoder, Registry};
 
-use exporter::{
+use actix_web::http::Uri;
+use clap::Parser;
+use exporter::pool::{
     cache,
     cache::{Cache, Data},
-    client::grpc_client::{GrpcClient, GrpcContext, Timeouts},
+    client::{
+        grpc_client::{GrpcClient, GrpcContext, Timeouts},
+        ApiVersion,
+    },
     collector::pools_collector::PoolsCollector,
     config::ExporterConfig,
     error::ExporterError,
 };
+use prometheus::{Encoder, Registry};
+use std::{env, net::SocketAddr, time::Duration};
+use tracing::{error, warn};
 
 /// Initialize exporter config that are passed through arguments
-fn initialize_exporter(args: &ArgMatches) {
-    ExporterConfig::initialize(args);
+fn initialize_exporter(args: &Cli) {
+    ExporterConfig::initialize(args.metrics_endpoint, args.polling_time.into());
 }
 
 /// Initialize mayastor grpc client
-async fn initialize_client() -> Result<GrpcClient, ExporterError> {
-    let t = Timeouts::new(Duration::from_millis(500), Duration::from_millis(500));
+async fn initialize_client(api_version: ApiVersion) -> Result<GrpcClient, ExporterError> {
+    let timeout = Timeouts::new(Duration::from_secs(1), Duration::from_secs(5));
```

Comment:
The timeout values are different?

Reply:
These are the default values we use in control-plane. But wanted to confirm were these values decided?

Reply:
> These are the default values we use in control-plane. But wanted to confirm were these values decided?

Ok, then we stick with the default values. AFAIK, the values weren't decided.

Reply:
Okay.